articles/policies/billing.md

```diff
@@ -88,15 +88,15 @@ Also, more than often the email which receives the receipt is the one that creat
 
 ## In our pricing, what is the difference between internal and external users? Are they different technically?
 
-An active user is a user that has authenticated with username/password, a passwordless connection or any social provider in the last 30 days, counted per application.
+An active user is a user that has authenticated with a username/password combination, a Passwordless connection, or any social provider in the last calendar month.
```

Comment:
@khorne3 should be *per* calendar month unstead of  *in the last* calendar month. 😬

Reply:
@solepano I don't think *per calendar month* works in this situation:

> An active user is a user that has authenticated with a username/password combination, a Passwordless connection, or any social provider per calendar month.

I think what you're looking for is *in a given calendar month*?

Reply:
👍 that would work, thanks!